src/Makevars

```diff
@@ -1,4 +1,2 @@
-
-## We want C++11 as it gets us 'long long' as well
-## If we uncomment this we fall back to C++98 and it is all tears
-CXX_STD = CXX11
+## Use C++11 where available. This ensures 'long long' (a C99 standard)
+PKG_CXXFLAGS=$(CXX1XSTD)
```

Comment:
Maybe I did not have enough coffee but how is that a fix, or how is that different from outright `CXX_STD=CXX11` ?

It is the exact same issue: rely on a newer compiler.